index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact">
+        <div class="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
```

Comment:
[OPTIONAL] Since this is a completely standalone `section` of our web app, I think you could use a `<section>` tag instead of a div to take care of the semantics of our code.

The same applies for the following `div.contact-header` which could be an `h` tag.

This is good for a range of things, going from having a cleaner and readable code, accessibility, and better SEO. :smiley: